### DISCUSSION TITLE
posts/vlan-firewall-unifi/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#242] [Comments]
# posts/vlan-firewall-unifi/

What is a VLAN and How Do They Help?

https://technotim.live/posts/vlan-firewall-unifi/

**Comment by Jeppedy** [comment]
Been a techie all my life (rude to ask me how long!), but still new to firewall rules/VLANs/Subnets/etc.
I followed your well-written directions, which make great sense, and yes, I cannot get from my IoT LAN to my Core LAN.  BUT!  I also seem to be having issues with SSH connection to my HomeAssistant RasPi (lives in IoT LAN) from Core.
Hangs as if the packets are being dropped, but works fine when I join my IoT network and SSH into it.

Do I need to add anything for permitting bi-directional flow?  Any hints to check would be gratefully accepted.
I'll do my diagnostics tomorrow, but I'm a bit spent tonight...

**Reply by Jeppedy** [reply]
Well, It would appear adding a firewall rule to ACCEPT from Core LAN to IoT LAN, enabling all "Match State" options, solved my issue.
Is it possible Unifi does NOT permit all, and an explicit ACCEPT is required now?

**Reply by Jeppedy** [reply]
Not sure if I overlooked a step, or if it isn't in the article, but the trick isn't adding the ACCEPT rule.  The trick us only applying the DROP rule to NEW traffic.  Otherwise, it snags the response traffic.  I made that change, and removed the ACCEPT rule, and it's all still working perfectly!

**Reply by timothystewart6** [reply]
Good call!

**Reply by RedPat** [reply]
Configuring the DROP rule to match to new traffic worked for me as well. 
In my case the first problem I encountered, was the Google Home App (Phone on trusted device VLAN) not able to connect to my Nest Hubs (IOT VLAN).
Anyways the tutorial was a huge help in setting up VLANs, thank you very much for that and your content in general! :)
